src/openrct2/interface/window.h

```diff
@@ -36,6 +36,7 @@ struct TitleSequence;
 struct TextInputSession;
 
 #define TEXT_INPUT_SIZE 1024
+#define TOP_TOOLBAR_CLEARANCE 28
```

Comment:
Ugh, fine, I'll create a define for the height (27) and adjust the `<` accordingly.